addons/web/static/tests/views/kanban_tests.js

```diff
@@ -685,19 +685,30 @@ QUnit.module('Views', {
     }),
 
     QUnit.test('delete a column in grouped on m2o', function (assert) {
-        assert.expect(26);
+        assert.expect(34);
```

Comment:
please write another test instead of modifying this one.  It is better to have 2 small unit tests that test 2 different part of a feature than 1 big test